packages/rfw/README.md

```diff
@@ -991,3 +991,34 @@ exclude that line from the coverage calculation.
 Golden tests are only run against the Flutter master channel and only
 run on Linux, since minor rendering differences are expected on
 different platforms and on different versions of Flutter.
+
```

Comment:
We should really move this whole Contributing section to a CONTRIBUTING.md, so it's not showing up on the pub.dev README, but that's out of scope here.

Reply:
Ah yes, that's a good call. I was tempted to move all the testing information to a README.md inside the `test` directory as well.

Reply:
Yikes, updating this README.md forces me to update the CHANGELOG. I'll add a `NEXT` entry, since these additional docs are more interesting for us than for the general public and might be moved.

Reply:
> We should really move this whole Contributing section to a CONTRIBUTING.md, so it's not showing up on the pub.dev README

Done, this way we have a good reason to properly publish the package as a z version.